docs/metadata-store/daily-use.md

```diff
@@ -14,9 +12,91 @@ and declare or delete resources such as queues or streams.
 As already mentioned in [clustering caveats](./clustering#caveats), Khepri is a
 Raft-based system, and just like in any Raft-based system, a quorum number of
 cluster members must be online and available for the metadata store to accept
-writes (inserts, updates, deletes).
+updates (writes/deletes or cluster membership changes).
+
+## Consistency Model and Visibility Guarantees
+
+Khepri has an important difference from Mnesia when it comes to schema
+modifications such as queue or stream declarations, or binding declarations.
+These changes won’t be noticeable with many workloads but can affect some, in
+particular, certain integration tests.
+
+### Example Scenarios
+
+Consider two scenarios, A and B.
+
+<figure>
+<div className={"figure"}>
+There is only one client. The client performs the following steps:
+
+1. It declares a queue Q.
+2. It binds Q to an exchange X.
+3. It publishes a message M to the exchange X.
+4. It expects the message to be routed to queue Q.
+5. It consumes the message.
+</div>
+<figcaption>Scenario A</figcaption>
+</figure>
+
+In this scenario, there should be no observable difference in behavior.
+Client’s expectations will be met.
+
+<figure>
+<div className={"figure"}>
+There are two clients, One and Two, connected to nodes R1 and R3, and using the
+same virtual host. Node R2 has no client connections.
+
+1. Client One declares a queue Q.
+2. It binds Q to an exchange X.
+3. It gets a queue declaration confirmation back.
```

Comment:
Should it be "binding declaration" instead of "queue declaration" ?

Reply:
Yes, it should be "binding declaration" or just "binding confirmation."